src/oidcop/token/id_token.py

```diff
@@ -249,7 +249,13 @@ def sign_encrypt(
 
         return _jwt.pack(_payload, recv=client_id)
 
-    def __call__(self, session_id: Optional[str] = "", ttype: Optional[str] = "", **kwargs) -> str:
+    def __call__(
+        self,
+        session_id: Optional[str] = "",
+        ttype: Optional[str] = "",
+        usage_rules: Optional[dict] = None,
```

Comment:
None or {}?

it's a dict, probably {} Is better

Reply:
I've always (well almost) refrained from using mutable values as defaults. So I'd vote for None.

Reply:
Yes, a dict for a default value is a bug waiting to happen

Reply:
I've always thought about why a variable of one type could take on another type, such as None. But it's a completely cosmetic thing in python, let's move on